packages/api-server/src/gasless/payload.ts

```diff
@@ -0,0 +1,46 @@
+import { HexNumber, HexString } from "@ckb-lumos/base";
+import { AbiCoder } from "web3-eth-abi";
+const abiCoder = require("web3-eth-abi") as AbiCoder;
```

Comment:
Override the previous import?

Reply:
the library `web3-eth-abi` does not works very well with typescript, here is a simple workaround